Formula/m/minder.rb

```diff
@@ -20,7 +20,7 @@ class Minder < Formula
   def install
     ldflags = %W[
       -s -w
-      -X github.com/stacklok/minder/internal/constants.CLIVersion=#{version}
+      -X github.com/mindersec/minder/internal/constants.CLIVersion=#{version}
```

Comment:
It seems this breaks the version reporting

Reply:
Hm, I wonder if that's because we haven't made a release yet from the new org.

Reply:
It's very likely that

Reply:
Alright, so we released a new version from the new org. I think I'll have to rebase the references here and bump to v0.0.66 too so it all aligns 👍

Reply:
hey @SMillerDev, here's what I did so far:
* We made a new release of minder - v0.0.66 - https://github.com/mindersec/minder/releases/tag/v0.0.66
* I ran `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source ./Formula/m/minder.rb`
```bash
➜  homebrew-core git:(patch-1) ✗ HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source ./Formula/m/minder.rb
Error: Failed to load cask: ./Formula/m/minder.rb
Cask 'minder' is unreadable: wrong constant name #<Class:0x0000000122c5c448>
Warning: Treating ./Formula/m/minder.rb as a formula.
==> Fetching minder
==> Downloading https://github.com/mindersec/minder/archive/refs/tags/v0.0.66.tar.gz
Already downloaded: /Users/dimitrovr/Library/Caches/Homebrew/downloads/76686239d76491debebdf62280bcb10699ff074d97c25a9511fd63590962ba39--minder-0.0.66.tar.gz
==> go build -ldflags=-s -w -X github.com/mindersec/minder/internal/constants.CLIVersion=0.0.66 ./cmd/cli
No config file present, using default values.
No config file present, using default values.
No config file present, using default values.
==> Caveats
zsh completions have been installed to:
  /opt/homebrew/share/zsh/site-functions
==> Summary
🍺  /opt/homebrew/Cellar/minder/0.0.66: 9 files, 36.2MB, built in 5 seconds
==> Running `brew cleanup minder`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```
* `brew test ./Formula/m/minder.rb`
```
➜  homebrew-core git:(patch-1) ✗ brew test ./Formula/m/minder.rb                                                      
==> Testing minder
==> /opt/homebrew/Cellar/minder/0.0.66/bin/minder version
No config file present, using default values.
==> /opt/homebrew/Cellar/minder/0.0.66/bin/minder artifact list -p github 2>&1
```
* `brew audit --strict minder` got nothing
```
➜  homebrew-core git:(patch-1) ✗ brew audit --strict minder
```

I still see the CI job failing for linux though and I also noticed this PR - https://github.com/Homebrew/homebrew-core/pull/194444 which fails in the same way.

Reply:
I'll resolve the conversation as the issue is no longer present 👍